### PR TITLE
Use label to discover Provisioned Service

### PR DESCRIPTION
**Pull request by baijum**
Signed-off-by: Baiju Muthukadan <baiju.m.mail@gmail.com>

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#docs) 
  included if any changes are user facing
- [ ] [Tests](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#tests)
  included if any functionality added or changed. For bugfixes please include tests that can catch regressions
- [ ] All acceptance test scenarios included in the PR which verifies a bugfix or a requested feature reported by a non-member are tagged with `@external-feedback` tag.
- [ ] Follows the [commit message standard](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#commits)

### CODE REVIEW

File: test/acceptance/features/supportExistingOperatorBackedServices.feature [hunk 1]
```diff
@@ -426,7 +426,6 @@ Feature: Support a number of existing operator-backed services out of the box
                 resource: deployments
           """
     Then Service Binding is ready
-    And Kind RabbitmqCluster with apiVersion rabbitmq.com/v1beta1 is listed in bindable kinds
```
Comment (baijum) [inline test/acceptance/features/supportExistingOperatorBackedServices.feature:429]: I have send a PR to add the label: https://github.com/rabbitmq/cluster-operator/pull/1165
For now, I have removed this test.

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/redhat-developer/servicebinding-operator
```
Comment (pmacik) [inline config/manager/kustomization.yaml:15]: Please, do not commit this file

**Comment by pmacik** [comment]
There is one HIGH security vulnerability recently identified (CVE-2022-32149), but is fixed already in master - please rebase to fix that.

**Pull request closed without merging**